datadog_checks_dev/datadog_checks/dev/terraform.py

```diff
@@ -20,6 +20,18 @@
     from shutilwhich import which
 
 
+def construct_env_vars():
+    # Terraform expects case-sensitive environment variables, which does not work inside tox
+    # on Windows since it passes down variables using the case-insensitive os.environ.
+    env = dict(os.environ)
+    for key in list(env):
+        if key.lower() == 'tf_var_account_json':
```

Comment:
Can you check `startswith('tf_var')` instead? That'll allow us to pass other variables later on.